tests/src/loader/Polyfill.test.ts

```
@@ -0,0 +1,39 @@
+    String.prototype.matchAll = null;
+    await import("@galacean/engine-loader");
+
+    const regexTest = /noGlobal/;
+    const contentTest = "test";
+    expect(
+      String.prototype.matchAll.bind(contentTest, regexTest),
+      "Should throw error caused by absence of g flag"
+    ).to.throw(TypeError);
+
+    const matchResult = content.matchAll(regex);
+    debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)